.github/workflows/build-and-publish.yml

```diff
@@ -27,12 +27,3 @@ jobs:
           POETRY_HTTP_BASIC_PYPI_USERNAME: "__token__"
           POETRY_HTTP_BASIC_PYPI_PASSWORD: "${{ secrets.PYPI_TOKEN }}"
         run: poetry publish
```

Comment:
Are you sure you don't want to publish slc-ci-setup releases to GitHub but still to Pypi?